site/soc/secrets/passphrases/osh_nova_metadata_proxy_shared_secret.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
for my understanding - does the master version **not** need this?

Reply:
yes the current master version as of today needs this too.

Reply:
Then splitting the commit would be nice. Then we could revert the "switch to master" commit (if we want to go back to master) but keep the fix